nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-core/src/main/java/org/nuxeo/connect/client/status/ConnectStatusHolder.java

```diff
@@ -54,9 +54,9 @@ protected ConnectRegistrationService getService() {
         return Framework.getLocalService(ConnectRegistrationService.class);
     }
 
-    public boolean isRegistred() {
+    public boolean isRegistered() {
```

Comment:
Let's discuss in [another thread](https://github.com/nuxeo/nuxeo/pull/873#discussion_r123192272).